patterns/wayfair-phone-otp.html

```diff
@@ -0,0 +1,22 @@
+<html gg-domain="wayfair">
+  <body>
+    <h1 gg-match="[data-test-id='AuthHeaderTitle']">Enter the Code We Texted You</h1>
+    <span gg-match="[data-test-id='READ_ONLY_PHONE']"></span>
+
+    <input type="text" name="otp" gg-match="input[name=otp]" placeholder="OTP" />
+    <button type="submit" gg-match="button[data-test-id='SIGN_IN_BUTTON']">Sign In</button>
+
+    <button
```

Comment:
We probably need to mimic the separator and the message "Other Ways to Sign In".